site/static/compare.html

```diff
@@ -340,7 +340,7 @@ <h2>Comparing <span id="stat-header">{{stat}}</span> between <span id="before">{
                 <div class="description-box">
                     <div v-if="data.prev" class="description-arrow"><a v-bind:href="prevLink">&larr;</a></div>
                     <div style="padding: 10px;">
-                        <a v-if="data.a.pr" v-bind:href="prLink(data.a.pr)">#{{data.a.pr}}</a>
+                        <a v-if="data.a.pr" v-bind:href="prLink(data.a.pr)">#{{data.a.pr}}&nbsp;</a>
```

Comment:
Should this be *inside* the link? Wouldn't that make the underline from links go under the space as well?

Reply:
Hmm, it's possible to botch even such a simple PR :D You're right, of course.

(Reminding myself that it would be nice to have an easier way of exporting data from the live site to offline storage, so that it's easier to check these things locally. I tried it with a hardcoded number and didn't notice it).